int16 Red
int16 Yellow
int16 Green
int16 Blue
int16 White
int16 Umber
---
int16 State